documentation/source/conf.py

```diff
@@ -42,8 +42,10 @@
 master_doc = 'contents'
 
 # General information about the project.
+import datetime
+year = datetime.datetime.now().year
 project = u'RMG Py'
-copyright = u'2002-2017, William H. Green, Richard H. West, and the RMG Team'
+copyright = u'2002-{0}, William H. Green, Richard H. West, and the RMG Team'.format(year)
```

Comment:
The license is read automatically from `LICENSE.txt`. I created a separate pull request (#1277) updating that file and all the file headers.

I can look into whether or not it's possible to prevent Sphinx from automatically doing syntax highlighting.